app/services/mastodon_tasks_service.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class MastodonTasksService
```

Comment:
This is sort of a weird name, because it's not really the same thing as the rest of the stuff in services.

I don't have a great idea for a better name/location ... maybe just `lib/maintenance.rb` or something boring?